patches/master_patch.patch

```diff
@@ -1597,7 +1610,7 @@ index cb7913b0a05f54ea405f4da4b2ca050db72814d3..7bc5de612ef4c9eaaf1c2335f40e38fc
 +  //                           url.SchemeIsWSOrWSS()));
```

Comment:
can you remove these changes? I think they're just whitespace, but I'd prefer to update them in a separate commit to avoid confusion